aten/src/THC/THCGeneral.h.in

```diff
@@ -64,6 +57,61 @@ typedef struct _THCCudaResourcesPerDevice {
   size_t scratchSpacePerStream;
 } THCCudaResourcesPerDevice;
 
+
+struct THCRNGState;  /* Random number generator state. */
+typedef struct CUDAStreamInternals THCStream;
+typedef struct THCState THCState;
+/* Global state of THC. */
```

Comment:
Why did you move this? Likely to conflict with #9735.

Reply:
Will move this back later if possible. Did this to deal with a forward declaration.